core/tests/protractor_utils/AdminPage.js

```diff
@@ -34,6 +34,64 @@ var AdminPage = function(){
   var roleSelect = element(by.css('.protractor-update-form-role-select'));
   var statusMessage = element(by.css('[ng-if="statusMessage"]'));
 
+  // The reload functions are used for mobile testing
+  // done via Browserstack. These functions may cause
+  // a problem when used to run tests directly on Travis.
+
+  var explorationElements = element.all(by.css(
+    '.protractor-test-reload-exploration-row'
+  ));
+
+  var reloadAllExplorationsButtons = element.all(by.css(
+    '.protractor-test-reload-all-explorations-button'
+  ));
+
+  var reloadCollectionButtons = element.all(by.css(
+    '.protractor-test-reload-collection-button'));
+
+  var explorationTitleElement = function(explorationElement) {
+    return explorationElement.element(
+      by.css('.protractor-test-reload-exploration-title')
+    );
+  };
+
+  var explorationElementReloadButton = function(explorationElement) {
+    return explorationElement.element(
+      by.css('.protractor-test-reload-exploration-button')
+    );
+  };
+
+  this.reloadCollection = function(collectionId) {
```

Comment:
Maybe have a config that's DEV_MODE and then in this file do:

```
    if (DEV_MODE) {
      var reloadAllExplorationsButtons = ...;
      ...
      this.reloadCollection = ...;
      ...
    }
```

since these are only relevant there.

Reply:
When --prod_env flag is used, the globals DEV_MODE variable is flipped.
1)The Activities tab in Admin Page is disable and replaced with the line :
`The 'Activities' tab is not available in the production environment.`

2)Also the little "Dev Mode" tag at the right corner of the page also disappears. 

You can take advantage of these GUI changes accordingly.

Reply:
Thanks for this -- quite helpful :)

Reply:
Done, thanks!

Reply:
Ah I was thinking of just declaring it similarly to what you have with browser.isMobile.

Reply:
Isn't this approach a bit better in the sense that we are actually checking for the dev mode? I tested it out and it seems to work fine although we can go the other way if you'd like :)

Reply:
You mean @hoangviet1993's approach? Yep, I'm fine with either; I was just answering your earlier question about "how do we access this variable here".

Reply:
Cool, understood! Thanks :)